linker.go

```diff
@@ -210,12 +210,6 @@ func fixupKfuncs(insns asm.Instructions) error {
 	return nil
 
 fixups:
-	// only load the kernel spec if we found at least one kfunc call
```

Comment:
Keep this here and pass the kernelSpec to the `findKfuncInKernel()` function.

Reply:
:white_check_mark: